### PR TITLE
[bitnami/clickhouse]: support clickhouse keeper

### LINKED ISSUE #13654: use clickhosue keeper replace zookeeper in chlickhosue chart
And, clickhouse will use [clickhouse keeper](https://clickhouse.com/docs/en/operations/clickhouse-keeper/) to replace zookeeper.

Official docs remove pre-production in [this commit](https://github.com/ClickHouse/ClickHouse/commit/fb08c5b7e14d9c5e54d8151f98f37192dd89c348) on Apr 15, 2022.. 

But this issue [In a degenerate scenario with concurrent writes, Keeper performance is about 10x lower than zookeeper](https://github.com/ClickHouse/ClickHouse/issues/41045#top) is not close.

_Originally posted by @yywing in https://github.com/bitnami/charts/issues/13541#issuecomment-1324511495_

### PR DESCRIPTION
**Pull request by yywing**
### Description of the change

Clickhouse chart support clickhouse keeper.

But still use zookeeper as defalut.

### Applicable issues

  - fixes #13654

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by corico44** [comment]
Hello @yywing, can you resolve the conflicts in the Chart.yaml and README.md files, please?

**Comment by yywing** [comment]
Done

**Comment by corico44** [comment]
Sorry @yywing, can you resolve the conflicts in the Chart.yaml and README.md files again and I will merge your changes, please?

**Comment by yywing** [comment]
Done

File: bitnami/clickhouse/values.yaml [hunk 1]
```diff
@@ -221,6 +228,12 @@ auth:
 ##
 logLevel: information
 
+## @section ClickHouse keeper configuration parameters
+## @param keeper.enabled Deploy ClickHouse keeper
```
Comment (corico44) [inline bitnami/clickhouse/values.yaml:232]: Can you modify the current text by "_Deploy ClickHouse keeper. Support is experimental._" and regenerate the README, please?

Comment (yywing) [reply bitnami/clickhouse/values.yaml:232]: ok

**Comment by corico44** [comment]
@yywing, can you resolve the conflicts in the README.md and Chart.yaml files?, please?

**Comment by yywing** [comment]
Done， aaaaaagain.

**Review by corico44 — APPROVED** [review]
LGTM

**Pull request merged**